src/main/haskell/matching-logic/matching-logic.cabal

```diff
@@ -0,0 +1,74 @@
+name:           matching-logic
```

Comment:
Because several important fields in `stack.yaml` cannot be generated from a `package.yaml` file.
There really isn't enough redundancy between `stack.yaml` and `<package>.cabal` to justify generating them from a common file in the first place.